plugin/mlsdbonlinelocator.cpp

```diff
@@ -276,11 +296,11 @@ QVariantMap MlsdbOnlineLocator::fallbackFields()
     // If no exact cell match can be found, fall back from exact cell position estimates to more
     // coarse grained cell location area estimates, rather than going directly to an even worse
     // GeoIP based estimate.
-    fallbacks["lacf"] = true;
+    fallbacks["lacf"] = false;
 
     // If no position can be estimated based on any of the provided data points, fall back to an
     // estimate based on a GeoIP database based on the senders IP address at the time of the query.
-    fallbacks["ipf"] = true;
+    fallbacks["ipf"] = false;
```

Comment:
Above comments are now exactly the wrong way?

Reply:
@dmamaevomp is it better to have no location at all than having a coarse location?

would it be useful to have something like this:
1) if we don't have any location yet, allow the ipf fallback, otherwise don't allow that fallback.
or
2) keep the ipf fallback AND if the location accuracy parameter is worse than something we acquired shortly before, don't use the inaccurate value
?

Reply:
We can put the accuracy parameter in the configuration settings, in dependency of value to set the on / off lpf fallback? ( do we have an external conf for geoclue-modules?)

Reply:
Initial comment was about comment in code saying it's going to fallback to more coarse positioning and then code actually disabling it. But agree with @krnlyng, coarse location should be better than no location at all. I'm not in favor of configuration, this should just do the right thing and give as good position as it can.

Reply:
Our office has the coordinates 55.739482, 37.507977. Service very often sent coordinates in the center of Moscow (55.752245, 37.615466), at the airport Sheremetyevo (55.966609, 37.413263), in the city of Krasnoyarsk (56.010964, 92.855856). I think the user does not want to have such false coordinates of the location. 
    I suggest one of the following options: 1) do not use false coordinates 2) add a filter with a choice of accuracy

Reply:
I can think of many cases when just getting the country right is better than nothing. Besides location includes the accuracy so either in your case the application doesn't know how to handle it or then we have problem here somewhere of accuracy not being set correct.

Reply:
In this case you need to add a new item "Low-accuracy positioning". This will save us from bad reviews about the work of the phone P4903

Reply:
I now checked: 
1) I form and send a request in which there is only one network "48:5b:39:d7:21:e1"
2) I received a reply QJsonDocument({"accuracy":1000,"location":{"lat":55.752200000000002,"lng":37.615600000000001}})
3) Service sent accuracy of 1000 meters, but in fact the accuracy is approximately equal to about 6500 meters
How can we determine the reliability of the data?